types.d.ts

```diff
@@ -1,3 +1,5 @@
+import type { ComponentType } from 'react'
```

Comment:
the 'react' may not exist since the `types.d.ts` is a remote module on deno.land, when you don't special the import maps

Reply:
fixed this with replacing to http link